### DISCUSSION TITLE
Enforce single team when self-hosting with v0.59

### DISCUSSION THREAD

**Discussion by borrinimarco74** [outline/outline#2581] [Self Hosting]
Hi folks,

I'm actually managing a self-hosted dockerized installation of Outline (version 0.59) that should be used as a migration of a Google site wiki.
A single container for a single wiki that should serve a single team made of:

- internal corporate users (let's say CORPORATE_EMAIL_DOMAIN)
- external users (let's say gmail.com + various EXTERNAL_EMAIL_DOMAIN)

under Google Authentication (for everyone, of course...)

So, after reading

- [https://github.com/outline/outline/pull/1954](url)
- [https://github.com/outline/outline/discussions/2404](url)

 I've defined

`ALLOWED_DOMAINS=CORPORATE_EMAIL_DOMAIN,gmail.com,EXTERNAL_EMAIL_DOMAIN`

and made a first login with a corporate email: everything is ok as it should be...

But for each other domain users (atm I can only check gmail.com users) I always get:

`Sorry, Google sign in cannot be used with a personal email. Please try signing in with your Google Workspace account.`

At this point, have I probably misunderstood this feature that should help me to reach my goal?

**Comment by engineer-myoa** [comment]
👍  +1 earnestly.
